src/points.jl

```diff
@@ -17,28 +17,35 @@ O = Point(0.0, 0.0) # origin of 2D Euclidean space
 
 ### Notes
 
-- Type aliases are `Point2`, `Point3`, `Point2f`, `Point3f`
+- Type aliases are `Point1`, `Point2`, `Point3`, `Point1f`, `Point2f`, `Point3f`
 """
 struct Point{Dim,T}
   coords::SVector{Dim,T}
+  Point{Dim,T}(coords::AbstractVector) where {Dim,T} = new{Dim,T}(coords)
```

Comment:
Just a quick question, could we also write this as follows?

```julia
Point{Dim,T}(coords) where {Dim,T} = new(coords)
```

I tend to omit the type annotation in inner constructors, I don't know if it makes a difference.

Reply:
Me too, but it is intentional here. The inner constructor has to have more specific type requirement than the varargs constructor so we don't run into the infinite recursion again.